package com.desafios.user.exception;

public record ErrorResponse(String message, int code) { }
